vendor/walletunlocker.proto

```diff
@@ -0,0 +1,338 @@
+syntax = "proto3";
+
+import "lightning.proto";
+
+package lnrpc;
```

Comment:
Package and go_package need to be changed

Reply:
Changed to `wallet_unlocker`?